### PR TITLE
Add org.small_tech.Gnomit

### PR DESCRIPTION
**Pull request by aral**
Hi there,

I’m migrating Gnomit from ind.ie.Gnomit to org.small_tech.Gnomit following the name change of our organisation from Ind.ie to Small Technology Foundation (https://small-tech.org).

The last version, at ind.ie.Gnomit was 1.1.0. This is 2.0.0, the first release with the new app ID.

Once I’ve submitted this, I will also update the previous app with an EOL indicator.

Thanks + lots of love from the Emerald Isle :)

### CODE REVIEW

**Comment by LeandroStanger** [comment]
bot, build org.small_tech.Gnomit

File: org.small_tech.Gnomit.appdata.xml [hunk 1]
```diff
@@ -0,0 +1,90 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
```
Comment (bilelmoussaoui) [inline org.small_tech.Gnomit.appdata.xml:2]: not a blocker, but the appdata/desktop files should go upstream

Comment (aral) [reply org.small_tech.Gnomit.appdata.xml:2]: Just to confirm, so should I simply remove the following files from the repo:

  - org.small_tech.Gnomit.appdata.xml
  - org.small_tech.Gnomit.desktop

Or is there something special I need to do to ensure these files “go upstream”. By upstream, I’m assuming you mean the app itself?

Happy to remove the files if they’re not necessary. 

Thanks!

Comment (LeandroStanger) [reply org.small_tech.Gnomit.appdata.xml:2]: New appdata files should use metainfo instead. https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html

Comment (x80486) [reply org.small_tech.Gnomit.appdata.xml:2]: @aral, what @bilelmoussaoui meant was that those files, ideally, should be part of your release; thus it is Gnomit should bundle them.

**Comment by barthalion** [comment]
I'll proceed with the merge, it doesn't make much sense to block it as it's just the rename. I'll open an issue with raised comments.

/merge

**Pull request closed without merging**